python-modules/cis_publisher/cis_publisher/mozilliansorg_groups.py

```diff
@@ -126,3 +125,25 @@ def _prepare_update(self, update):
             logger.critical("Profile data signing failed for user {}: {}".format(update.user_id, e))
             return None
         return update_profile
+
+    @staticmethod
+    def _update_groups(current_groups, mozillians_groups_update):
+        """
+        Update current gropus based on a mozillians groups update. PMO groups (value == "") overwrite mozillians groups.
+
+        @current_groups: current groups dict from profile v2
+        @mozillians_gropus_update: list of mozillians groups
+        @return: new groups dict or None if nothing changed
+        """
+        if mozillians_groups_update is None:
+            return None
+        updated = {group: None for group in mozillians_groups_update}
```

Comment:
Can you help me understand why it's `group: None` like this?

Reply:
Access groups are an object in the profile v2 schema. We never defined how it should look. Right now all access groups are `{ key: null, … }` (ldap/hris/mozilliansorg). To distinguish PMO and Mozillians managed access group during migration the ones managed by PMO will be `{ key: something }` to not override each other and for easy migration.